docs/reference/tags/form_tags/clear_cart/index.md

```diff
@@ -0,0 +1,47 @@
+---
+layout: default
+title: Clear Cart
+parent: Tags
+has_children: false
+---
+
+The Clear Cart Tag renders a button that clears the user's cart. 
+
+##### input
+{% raw %}
+```liquid
+{% form "clear_cart", return_to: config.clear_cart_link.url %}
+	 <input type="submit" value="clear" class="cart__clear {{id}}__clear" />
```

Comment:
Given those classes aren't rendered by the tag, and aren't important for this to work, I'd remove it.